Makefile

```diff
@@ -1,4 +1,5 @@
-.PHONY: server
+.PHONY: cmd components server
```

Comment:
Is this change expected? There is no target named `cmd`.

Reply:
There is a directory `cmd`, so if run `make cmd`, it will try to make the directory. We don't have any Makefile in `cmd` so it's not a problem for now.